sdks/python/apache_beam/transforms/trigger.py

```diff
@@ -260,6 +262,55 @@ def to_runner_api(self, unused_context):
         default=beam_runner_api_pb2.Trigger.Default())
 
 
+class AfterProcessingTime(TriggerFn):
+  """Fire exactly once after a specified delay from processing time.
+
+  AfterProcessingTime is experimental. No backwards compatibility guarantees.
+  """
+
+  def __init__(self, delay=0):
+    self.delay = delay
+
+  def __repr__(self):
+    return 'AfterProcessingTime(delay=%d)' % self.delay
+
+  def on_element(self, element, window, context):
+    context.set_timer(
+        '', TimeDomain.REAL_TIME, context.get_current_time() + self.delay)
+
+  def on_merge(self, to_be_merged, merge_result, context):
+    # timers will be kept through merging
```

Comment:
<!--new_thread; commit:a68ceca8046b6d42f941aa4016a528ec2ee3808a; resolved:0-->
Please add capitalization, period at end of sentence.

Reply:
I was following the same convention as in `AfterCount`'s on_merge, since it isn't a docstring, but just a comment.